examples/tfengine/generated/multi_envs/devops/terraform.tfvars

```diff
@@ -26,10 +29,18 @@ project = {
     "cloudidentity.googleapis.com",
   ]
   owners_group = {
-    id           = "example-devops-owners@example.com"
-    display_name = "example-devops-owners"
     customer_id  = "c12345678"
-    owners       = ["user1@example.com"]
+    description  = ""
+    display_name = "example-devops-owners"
+    exists       = false
+    id           = "example-devops-owners@example.com"
+    owners = [
+      "user1@example.com",
```

Comment:
This might not work with. The `,` at the end might cause issue

Reply:
Tested with these 3 different combinations on a cicd recipe. They all work.

I'm resolving

![Screenshot 2021-08-11 11 57 03 AM](https://user-images.githubusercontent.com/33379285/129071744-27aa2f8d-2dd5-4e1d-818c-60c21529bff2.png)
![Screenshot 2021-08-11 11 59 54 AM](https://user-images.githubusercontent.com/33379285/129071742-669b8744-86d7-4e37-af50-c8d133a7652c.png)
![Screenshot 2021-08-11 12 00 38 PM](https://user-images.githubusercontent.com/33379285/129071735-eed119b1-eb71-4bd7-85ef-6f637fd410cb.png)